mdpow/config.py

```diff
@@ -112,15 +112,17 @@
 
 def merge_dicts(user, default):
     """Merge two dictionaries recursively.
-
-    Based on https://stackoverflow.com/a/823240/334357
+    Uses recursive method to accurately
+    merge nested dictionaries
     """
-    if isinstance(user, dict) and isinstance(default, dict):
-        for k, v in default.iteritems():
-            if k not in user:
-                user[k] = v
-            else:
-                user[k] = merge_dicts(user[k], v)
+    for key in default:
```

Comment:
Replaces previous method to remove dependency on `iteritems()`

Reply:
I think using `items()` would have been ok but the new one is fine with me, too.